src/v3/src/components/Spinner/Spinner.tsx

```diff
@@ -30,6 +30,7 @@ const Spinner: FunctionComponent<{
       flexDirection="column"
       justifyContent="center"
       alignItems="center"
+      data-se="o-spinner"
```

Comment:
Should the data-se go on the Box or on `CircularProgress` directly?

Reply:
this one was added to verify loading state in jest, not for testcafe parity. I am ok with either way.

Reply:
I thought you added this for a testcafe test. If this is only for jest, then update the test rather than add a `data-se` if possible.

Reply:
we will need a selector as id, that's what suggested by testing library. also, what does an extra data-se hurt?

Reply:
> also, what does an extra data-se hurt?

The added `data-se` isn't necessary. Why add it?

We can use `ByLabelText` https://testing-library.com/docs/queries/bylabeltext

RTL docs recommend using `ByTestId` as the last resort, so if we have an option to query by something else we should make an effort to do so: https://testing-library.com/docs/queries/about/#priority

Reply:
does a spinner have a label text?

Reply:
Yes, `aria-label` is counted, see the RTL docs linked above.